packages/opentelemetry-resources/src/detectors/BrowserDetector.ts

```diff
@@ -61,5 +62,18 @@ class BrowserDetector implements Detector {
   }
 }
 
+//Add Browser related attributes to resources
+function getBrowserAttributes():ResourceAttributes {
+  const browserAttribs : ResourceAttributes = {};
+  const userAgentData : UserAgentData = (navigator as any).userAgentData;
+  if(userAgentData) {
+    browserAttribs['browser.platform'] = userAgentData.platform;
+    browserAttribs['browser.brands'] = userAgentData.brands;
+    browserAttribs['browser.mobile'] = userAgentData.mobile;
+  }
+  browserAttribs['browser.user_agent'] = navigator.userAgent;
```

Comment:
Regarding the semantic convention attributes, we can create a constant map for it and add a comment that it should be replaced once the semantic convention package is updated. So that we don't need to repeat these constants here and in tests.

Reply:
Check if the change is as you requested. Also we have redundant attributes in version and user_agent. What do you suggest ?